modules/sdk-coin-apt/src/lib/transaction/digitalAssetTransaction.ts

```diff
@@ -0,0 +1,73 @@
+import { Transaction } from './transaction';
+import {
+  AccountAddress,
+  Aptos,
+  AptosConfig,
+  EntryFunctionABI,
+  MoveAbility,
+  Network,
+  objectStructTag,
+  TransactionPayload,
+  TransactionPayloadEntryFunction,
+  TypeTagAddress,
+  TypeTagGeneric,
+  TypeTagStruct,
+} from '@aptos-labs/ts-sdk';
+import { InvalidTransactionError, TransactionRecipient, TransactionType } from '@bitgo/sdk-core';
+import { BaseCoin as CoinConfig, NetworkType } from '@bitgo/statics';
+import { DIGITAL_ASSET, DIGITAL_ASSET_FUNCTION, DIGITAL_ASSET_TRANSFER_SUPPLY_PER_TRANSACTION } from '../constants';
+
+export class DigitalAssetTransaction extends Transaction {
+  constructor(coinConfig: Readonly<CoinConfig>) {
+    super(coinConfig);
+    this._type = TransactionType.SendNFT;
+    this._assetId = AccountAddress.ZERO.toString();
+  }
+
+  protected parseTransactionPayload(payload: TransactionPayload): void {
+    if (
+      !(payload instanceof TransactionPayloadEntryFunction) ||
+      payload.entryFunction.args.length !== 2 ||
+      DIGITAL_ASSET !== payload.entryFunction.type_args[0].toString()
+    ) {
+      throw new InvalidTransactionError('Invalid transaction payload');
+    }
+    const entryFunction = payload.entryFunction;
+    if (!this._recipient) {
+      this._recipient = {} as TransactionRecipient;
+    }
+    this._assetId = entryFunction.args[0].toString();
+    this._recipient.address = entryFunction.args[1].toString();
+    this._recipient.amount = DIGITAL_ASSET_TRANSFER_SUPPLY_PER_TRANSACTION;
+  }
+
+  protected async buildRawTransaction(): Promise<void> {
+    const network: Network = this._coinConfig.network.type === NetworkType.MAINNET ? Network.MAINNET : Network.TESTNET;
+    const aptos = new Aptos(new AptosConfig({ network }));
+    const senderAddress = AccountAddress.fromString(this._sender);
+    const recipientAddress = AccountAddress.fromString(this._recipient.address);
+    const digitalAssetAccountAddress = AccountAddress.fromString(this._assetId);
```

Comment:
for consistency with other similar variables.
```suggestion
    const digitalAssetAddress = AccountAddress.fromString(this._assetId);
```

Reply:
done